src/function.ts

```diff
@@ -950,19 +1011,28 @@ export async function serialize(
            */
           const transformIntegration = (integ: unknown): any => {
             if (integ && isIntegration(integ)) {
-              const copy = {
-                ...integ,
-                native: {
-                  call: integ?.native?.call,
-                  preWarm: integ?.native?.preWarm,
-                },
-              };
-
-              INTEGRATION_TYPE_KEYS.filter((key) => key !== "native").forEach(
-                (key) => delete copy[key]
-              );
-
-              return copy;
+              const c = integ.native?.call;
+              const call =
+                typeof c !== "undefined"
+                  ? function (...args: any[]) {
+                      return c(args, preWarmContext);
```

Comment:
I love this bootstrap of preWarmContext

Reply:
I think this means we don't have to rewrite ast to accept prewarm context anywhere now right?

Reply:
yep, no more native function specific code in compile (except for transforming the closure to ast)